src/main.cpp

```diff
@@ -299,9 +336,22 @@ int main(int argc, char* argv[])
         {
             resetFirmware(interactive, forceFlash);
         }
+#ifdef INTEL_PLATFORMS
+        else if (!nvramReadFile.empty())
+        {
+            BIOSUpdater::readNvram(nvramReadFile);
+        }
+        else if (!nvramWriteFile.empty())
+        {
+            BIOSUpdater::writeNvram(nvramWriteFile);
+        }
+#endif
         else
         {
             fprintf(stderr, "One or both of --file/--reset "
+#ifdef INTEL_PLATFORMS
+                            "or one of --gbe/--nvread/-nvwrite "
```

Comment:
as far as I can see, I can use __both__ `-f filename1` and `-g filename2`, but only the filename from the option used last will be used. I don't see why the `-g` option requires an argument. It's apparenly just a modifier flag, like `-r`.

Reply:
I saw nothing bad in that case. `-g path` was a sub command that could not be specified with `-f` at the same call.
Current implementation (`-g` is a flag for the flash sub command) looks also well.